File: java/org.financer.client.javafx/src/main/java/org/financer/client/javafx/main/ApplicationLauncher.java
package org.financer.client.javafx.main;

import javafx.application.Application;

public class ApplicationLauncher {
    public static void main(String[] args) {
        System.setProperty("javafx.preloader", "org.financer.client.javafx.main.SplashScreenLoader");
        Application.launch(FinancerApplication.class, args);
    }
}

File: java/org.financer.client.javafx/src/main/kotlin/org/financer/client/javafx/main/ApplicationLauncher.kt
package org.financer.client.javafx.main

import javafx.application.Application

object ApplicationLauncher {
    @JvmStatic
    fun main(args: Array<String>) {
        System.setProperty("javafx.preloader", "org.financer.client.javafx.main.SplashScreenLoader")
        Application.launch(FinancerApplication::class.java, *args)
    }
}